v1.1/admin-ui-access-and-navigate.md

```diff
@@ -0,0 +1,103 @@
+---
+title: Access and Navigate CockroachDB Admin UI
+toc: false
+---
+
+<div id="toc"></div>
+
+## Access the Admin UI
+
+You can access the Admin UI from any node in the cluster.
+
+By default, you can access it via HTTP on port `8080` of the hostname or IP address you configured using the --host flag while [starting the node](https://www.cockroachlabs.com/docs/stable/start-a-node.html#general). For example, `http://<any node host>:8080`. If you are running a secure cluster, use `https://<any node host>:8080`.
+
+You can also set the CockroachDB Admin UI to a custom port using `--http-port` or a custom hostname using `--http-host` when [starting each node](start-a-node.html) (i.e., each node's values are dependent on how it's started; there is no cluster-level configuration for non-default values). For example, if you set both a custom port and hostname, `http://<http-host value>:<http-port value>`. For a secure cluster, `https://<http-host value>:<http-port value>`.
```

Comment:
This predates you, but let's remove the parenthetical: `(i.e., each node's values....)`